0-ad-history-mod-package/simulation/templates/gaia/fauna_chicken.xml

```diff
@@ -13,8 +13,8 @@
     <Icon>gaia/fauna_chicken.png</Icon>
     <History>
         Bred from the jungle fowl in Southeast Asia, the chicken (Gallus gallus domesticus) is a very common and important farm animal. Farmers since ancient times been raised for both its eggs and, when the bird grew too old to lay, for meat. However, also Chickens were raised for cock fighting which was a very popular sport.
-         It is even written that Themistocles while he was leading his troops against the persians stopped so that his soldiers could watch the fight. He then said. “Behold, these do not fight for their household gods, for the monuments of their ancestors, for glory, for liberty or the safety of their children, but only because one will not give way to the other." And thus inspired they went on to defeat the persians.
-        Chickens come in all sizes but usually weigh around 7 to 10 pounds. They are foraging ground birds that can only fly for short distances.
+         It is even written that Themistocles while he was leading his troops against the Persians stopped so that his soldiers could watch the fight. He then said. “Behold, these do not fight for their household gods, for the monuments of their ancestors, for glory, for liberty or the safety of their children, but only because one will not give way to the other." And thus inspired the soldiers went on to defeat the Persians.
```

Comment:
```suggestion
         It is even written that Themistocles, while he was leading his troops against the Persians, stopped so that his soldiers could watch the fight. He then said: “Behold, these do not fight for their household gods, for the monuments of their ancestors, for glory, for liberty or the safety of their children, but only because one will not give way to the other." And thus inspired the soldiers went on to defeat the Persians.
```